compiler/semmagic.nim

```diff
@@ -128,19 +132,27 @@ proc evalTypeTrait(traitCall: PNode, operand: PType, context: PSym): PNode =
     return typeWithSonsResult(tyNot, @[operand])
   of "name":
     result = newStrNode(nkStrLit, operand.typeToString(preferTypeName))
-    result.typ = newType(tyString, context)
+    result.typ = newType(tyString, owner)
     result.info = traitCall.info
+  of "vtptr", "vtref":
+    debug operand
```

Comment:
Don't forget to remove this.

Reply:
This is still work-in-progress. It's not ready for merging yet.

Reply:
The bugfixes are blocking the release though.

Reply:
Don't pressure poor @zah :)